### DISCUSSION TITLE
更改安装位置后视频缓冲位置

### DISCUSSION THREAD

**Discussion by Alex-syz** [Richasy/Bili.Uwp#615] [问答]
由于我的C盘太小了，就把哔哩移动到了另一个盘里，然后就出现了个玄学的现象，放视频的时候明明流量都跑到两三百兆了但是没向任何一个盘里缓冲数据，（还是任务管理器无法检测？）
有没有好兄弟能解答一下，也恳请大佬能加个自定义缓冲数据位置的选项，毕竟C盘太小真的伤不起。

**Comment by hez2010** [comment]
可能缓冲放到内存中没有落盘

**Comment by Alex-syz** [comment]
谢谢您的解答，缓存进入内存那简直太棒了

Steve ***@***.***> 于 2021年11月19日周五 上午9:12写道：
